xml/System.Runtime.InteropServices/ComRegisterFunctionAttribute.xml

```diff
@@ -42,9 +42,11 @@
 ## Remarks  
  You can apply this attribute to methods.  
   
- <xref:System.Runtime.InteropServices.ComRegisterFunctionAttribute> enables you to add arbitrary registration code to accommodate the requirements of COM clients. For example, you can update the registry using registration functions from the <xref:Microsoft.Win32> namespace. If you provide a registration method, you must also apply <xref:System.Runtime.InteropServices.ComUnregisterFunctionAttribute?displayProperty=nameWithType> to an unregistration method, which reverses the operations done in the registration method.  
+ <xref:System.Runtime.InteropServices.ComRegisterFunctionAttribute> enables you to add arbitrary registration code to accommodate the requirements of COM clients. For example, you can update the registry using registration functions from the <xref:Microsoft.Win32> namespace. If you provide a registration method, you should also apply <xref:System.Runtime.InteropServices.ComUnregisterFunctionAttribute?displayProperty=nameWithType> to an unregistration method, which reverses the operations done in the registration method.  
```

Comment:
```suggestion
 `ComRegisterFunctionAttribute` enables you to add arbitrary registration code to accommodate the requirements of COM clients. For example, you can update the registry using registration functions from the <xref:Microsoft.Win32> namespace. If you provide a registration method, you should also apply <xref:System.Runtime.InteropServices.ComUnregisterFunctionAttribute?displayProperty=nameWithType> to an unregistration method, which reverses the operations done in the registration method.  
```
In the current build system, this xref to the current type creates a self-referencing link which, when clicked, causes a page reload.